code/modules/admin/verbs/randomverbs.dm

```diff
@@ -754,6 +676,9 @@ Traitors and the like can also be revived with the previous role mostly intact.
 	set name = "Gibself"
 	set category = "Event"
 
+	if(!check_rights(R_ADMIN|R_EVENT))

```

Comment:
Does this mean you need to have R_ADMIN and R_EVENT, or R_ADMIN or R_EVENT, for clarification?